src/Header/AcceptHeader.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+/**
+ * Spiral Framework.
+ *
+ * @license   MIT
+ * @author    Pavel Z
+ */
+
+declare(strict_types=1);
+
+namespace Spiral\Http\Header;
+
+/**
+ * Represents "Accept" header item.
+ */
+class AcceptHeader
+{
+    /** @var AcceptHeaderItem[] */
+    private $items = [];
```

Comment:
The constructor overrides `array` default value.
```suggestion
    private $items;
```